tests/functional/test_get_coordinates.py

```diff
@@ -0,0 +1,22 @@
+from app.models import Cities
+from geocoding_v1 import get_coordinates, update_coordinates_db
+
+
+def test_get_coordinates(client, insert_data_db):
+    '''
+    GIVEN a function to call on Google Geolocat API to get the coordinates
+    WHEN this functions is called on with the city 'Amsterdam'
+    THEN check that the returned coordinates are as expected and are stored in the DB
+    '''
+    city = 'Amsterdam'
+    city_query = Cities.query.filter_by(city_name=city).first()
+    coordinates = get_coordinates(city_query.city_name)
```

Comment:
Since `get_coordinates` calls an external API it might fail or take too long to respond. 
This kind of test might be `flaky` - most of the times it passes, but sometimes it doesn't
One way to solve this is to **mock** `get_coordinates` to return a predictable result
Or to mock that it raises an Exception (when you want to test the unhappy path)

This "mocking" is a typical aspect of integration tests - test where you want to test a chunk of the system but not all of it. I find these tests more challenging to write because you need to decide if / when / how to mock and still get a test that is realistic.